tests/scf-cholesky-basis/input.dat

```diff
@@ -15,16 +15,17 @@ e_canonical   = -230.64152543331906
 # indices. Since there's no guarantee that the same functions get
 # picked every time, we need to allow for changes in the energy that
 # go to zero when the tolerance is tightened.
+# MKL 2020.1 seems to often pick different fns, so need to relax the tolerance from 1.e-8 to 2.e-8
 
 tol_4 = 3
 tol_5 = 4
 tol_6 = 5
-tol_7 = 7
-tol_8 = 7
-tol_9 = 7
-tol_10 = 8
-tol_11 = 8
-tol_can = 8
+tol_7 = 2.e-8
+tol_8 = 2.e-8
+tol_9 = 2.e-8
```

Comment:
Shouldn't these be 2e-7?

Reply:
They were all 1.e-8 before yesterday.
https://github.com/psi4/psi4/commit/ac1bd292bc711b78ff593a73027c8071a42d86f6#diff-db0b071835a8a6a10a315bb623c173cb  All the differences are about 1.14e-8. Jonathon relaxed tol_7/8/9 to 1.e-7 yesterday, but I think the lot works fine with the more conservative 2.e-8. So I think as-is is right.